api/src/pcapi/core/users/models.py

```diff
@@ -930,3 +935,21 @@ class UserProNewNavState(PcObject, Base, Model):
     eligibilityDate: datetime = sa.Column(sa.DateTime, nullable=True)
     # If set, it means that the user has switched to the new nav at that date. Otherwise, it means that they are still on the old nav
     newNavDate: datetime = sa.Column(sa.DateTime, nullable=True)
+
+
+class GdprDataExtractUser(PcObject, Base):
+    __tablename__ = "gdpr_data_extract_user"
```

Comment:
Ne serait-il pas plus compréhensible d'avoir « user data », plutôt que « user » à la fin ne donnant pas une expression claire ?
```suggestion
class GdprUserDataExtract(PcObject, Base):
    __tablename__ = "gdpr_user_data_extract"
```